plugins/inputs/mqtt_consumer/mqtt_consumer.go

```diff
@@ -146,6 +151,11 @@ func (m *MQTTConsumer) Init() error {
 		}
 	}
 
+	tags := map[string]string{
+		"address": m.Servers[0],
```

Comment:
It seems a bit strange to only add the first configured server as tag. I would either create a stat for each or just all together without the mention of the specific server.

Reply:
Do you recommend making the tag something like the topic_tag, or maybe the ClientID?

Reply:
I think the original requestor in #11334 only needed a global byte counter, so I would add no tags.